odoo/addons/base/models/res_users.py

```diff
@@ -701,7 +701,7 @@ def context_get(self):
     def _get_company_ids(self):
         # use search() instead of `self.company_ids` to avoid extra query for `active_test`
         domain = [('active', '=', True), ('user_ids', 'in', self.id)]
-        return frozenset(self.env['res.company'].search(domain).ids)
+        return tuple(self.env['res.company'].search(domain).ids)
```

Comment:
-> `self.env['res.company'].search(domain)._ids`

Reply:
How is it different? I don't want to change anything more than needed :/